railties/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Add internal route `/rails/info/notes`, that displays the same information as `bin/rails notes`.
```

Comment:
I'm not sure `route` is an obvious term here.
There's more that just a route (like the HTML page) and this page displays the information.
Maybe `path` or `page`?
```suggestion
*   Add internal page `/rails/info/notes`, that displays the same information as `bin/rails notes`.
```